source/module_base/math_integral.cpp

```diff
@@ -384,4 +384,71 @@ void Integral::Gauss_Legendre_grid_and_weight(const double xmin, const double xm
 
 }
 
+const double Integral::Lebedev_Laikov_grid110_x[110] = {
```

Comment:
There are some constant lists, please add annotations to describe where are these lists from.